languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/instructions.md

```diff
@@ -65,24 +65,24 @@ Define the `remove_the_mean_person()` function that takes 2 parameters `queue, p
 2. person_name is the name of the person whom we need to kick out.
 
 ```python
-remove_the_mean_person(queue=["Natasha", "Steve", "Eltran", "Wanda", "Rocket"], person_name="Eltran")
-#=> ["Natasha", "Bucky", "Steve", "Wanda", "Rocket"]
+>>> remove_the_mean_person(queue=["Natasha", "Steve", "Eltran", "Wanda", "Rocket"], person_name="Eltran")
+["Natasha", "Steve", "Wanda", "Rocket"]
 ```
 
-# 5. DoppelGangers
+# 5. Namefellows
 
-You may not have seen 2 unrelated people who look exactly the same, but you've _definitely_ seen unrelated people with the same name exact name! It looks like today there is a lot of that happening. You want to know how many times a particular name has occured in the queue.
+You may not have seen 2 unrelated people who look exactly the same, but you've _definitely_ seen unrelated people with the same name exact name! It looks like today there is a lot of that happening. You want to know how many times a particular name has occurred in the queue.
 
-Define the `how_many_dopplegangers()` function that takes 2 parameters `queue, person_name`.
```

Comment:
`dopplegangers` should be written `doppelgangers` but I changed it to `namefellows` that I find more appropriate and poetic.